x-pack/solutions/observability/plugins/slo/public/pages/slos/components/health_callout/health_callout.tsx

```diff
@@ -83,69 +74,25 @@ export function HealthCallout({ sloList }: { sloList: SLOWithSummaryResponse[] }
           <EuiFlexItem>
             <FormattedMessage
               id="xpack.slo.sloList.healthCallout.description"
-              defaultMessage="The following {count, plural, one {transform is} other {transforms are}
-          } in an unhealthy state:"
+              defaultMessage="The following {count, plural, one {SLO is} other {SLOs are}}
+          in an unhealthy state. Data may be missing or incomplete. You can inspect {count, plural, one {it} other {each one}} here:"
               values={{
-                count: unhealthyRollupTransforms.length + unhealthySummaryTransforms.length,
+                count: unhealthySloList.length,
               }}
             />
             <ul>
-              {unhealthyRollupTransforms.map((result) => (
-                <li>
-                  {getSLOTransformId(result.sloId, result.sloRevision)}
-                  <EuiCopy textToCopy={getSLOTransformId(result.sloId, result.sloRevision)}>
-                    {(copy) => (
-                      <EuiButtonIcon
-                        data-test-subj="sloHealthCalloutCopyButton"
-                        aria-label={i18n.translate(
-                          'xpack.slo.sloList.healthCallout.copyToClipboard',
-                          { defaultMessage: 'Copy to clipboard' }
-                        )}
-                        color="text"
-                        iconType="copy"
-                        onClick={copy}
-                      />
-                    )}
-                  </EuiCopy>
-                </li>
-              ))}
-
-              {unhealthySummaryTransforms.map((result) => (
-                <li>
-                  {getSLOSummaryTransformId(result.sloId, result.sloRevision)}
-                  <EuiCopy textToCopy={getSLOSummaryTransformId(result.sloId, result.sloRevision)}>
-                    {(copy) => (
-                      <EuiButtonIcon
-                        data-test-subj="sloHealthCalloutCopyButton"
-                        aria-label={i18n.translate(
-                          'xpack.slo.sloList.healthCallout.copyToClipboard',
-                          { defaultMessage: 'Copy to clipboard' }
-                        )}
-                        color="text"
-                        iconType="copy"
-                        onClick={copy}
-                      />
-                    )}
-                  </EuiCopy>
+              {unhealthySloList.map((result) => (
+                <li key={getSLOTransformId(result.sloId, result.sloRevision)}>
```

Comment:
Now we can probably use just the `sloId` here 😃

We don't need the transform id if we have one element per SLO.